interpreter.ts

```diff
@@ -26,6 +26,14 @@ module J2ME {
     traceWriter.writeLn(toDebugString(array) + "[" + index + "] (" + toDebugString(array[index]) + ")");
   }
 
+  function classInitAndUnwindCheck(classInfo: ClassInfo, pc: number) {
+    classInitCheck(classInfo);
+    if (U) {
```

Comment:
classInitCheck is already doing this for all but array classes.
Since classInitCheck is only used here, I think inlining it would make classInitAndUnwindCheck clearer.